gitbutler-ui/src/lib/branches/service.ts

```diff
@@ -68,11 +68,21 @@ export class BranchService {
 
 		const createPrSpan = sentryTxn.startChild({ op: 'pr_api_create' });
 
+		let title = newBranch.name;
+		let body = newBranch.notes;
+
+		// In case of a single commit, use the commit summary and description for the title and
+		// description of the PR
+		if (newBranch?.commits?.length === 1) {
```

Comment:
@eyalch Why do you suggest to remove the chaining operator? Especially the `newBranch?.commits` one because `newBranch` has the type `Branch | undefined`

Reply:
There's already a check that throws an error in case `newBranch` is `undefined`:
https://github.com/gitbutlerapp/gitbutler/blob/065e3264e625dc75d9eed977606d2270e4941cc9/gitbutler-ui/src/lib/branches/service.ts#L59-L63

Also, if you glance 2 lines above, you'll see that we already rely on `newBranch` having a value, which means that, if `newBranch` were `undefined` in some case, this optional chaining is already too late and we'd have got an error.

Reply:
0kay. Makes sense. Thank you.

Reply:
Resolving the comment now.